### DISCUSSION TITLE
cdn for vcpkg?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vcpkg#28500] [Q&A] [answered]
Hi everyone.

*Would it be possible to use cdn for vcpkg?*

**Comment by JackBoosY** [comment, marked as answer]
Currently vcpkg only provides the build process, not the binaries.

**Reply by ghost** [reply]
Hi JackBoosY.

> Currently vcpkg only provides the build process, not the binaries.

thank you for feedback here.
